QUEST_JOBSTEP.tsv

```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
 QUEST_JOBSTEP_20150317_000061	Huh, this is the tool I lost. {nl}Thank you. Where has it been?
 QUEST_JOBSTEP_20150317_000062	Did you forget about the donation? {nl}It's 5000 silver.
 QUEST_JOBSTEP_20150317_000063	Cleric Master
-QUEST_JOBSTEP_20150317_000064	Thank you in the name of Goddess.{nl}I promise the donations will be used for those injured by monsters. {nl}
-QUEST_JOBSTEP_20150317_000065	I used to be a thief, but I came to realize certain things and I stopped my life of crime. {nl}But there is a little load left on my mind...and I will resolve it now {nl}
+QUEST_JOBSTEP_20150317_000064	$Thank you, in the name of the Goddess.{nl}I promise the donations will be used for those injured by monsters. {nl}
+QUEST_JOBSTEP_20150317_000065	$I used to be a thief, but I came to realize certain things and I stopped my life of crime. {nl}But there is a little load left on my mind...and I will resolve it now. {nl}
```

Comment:
Missing a space after the ellipse. 
I'd also write this as "But there is still a small load left on my mind... and I will get rid of it now."